### ISSUE TITLE
new type inference: incorrect warning about disabled attribute of the fieldset element

### ISSUE THREAD

**Issue by ghosttie** [google/closure-compiler#1449]
I'm getting this warning:

```
filevision/ui/field/TabField.js:218: WARNING - Property disabled never defined on HTMLFieldSetElement
    return (/** @type {!HTMLFieldSetElement} */ (this.element)).disabled;
           ^
```

But the disabled property looks like it's part of [the standard](https://html.spec.whatwg.org/multipage/forms.html#attr-fieldset-disabled) and according to [caniuse ](http://caniuse.com/#feat=fieldset-disabled) is well supported.

**Comment by MatrixFrog** [comment]
It's just missing from our externs, it looks like.

**Comment by concavelenz** [comment]
Care to send a PR?

**Comment by ghosttie** [comment]
I'd be happy to, but I've never done that before, I'll look up how to

**Comment by concavelenz** [comment]
The basic definition is here:
https://github.com/google/closure-compiler/blob/da60cd0ef7ac0e0cf2fd4181175593430772bc2d/externs/browser/w3c_dom2.js#L1197

**Comment by dimvar** [comment]
closing since this is an externs issue, not an NTI issue
